README.md

```
@@ -291,6 +349,16 @@ For non-Pacs002 flows, `TxTp`, `TenantId`, `MsgId`, and `Payload` are required.
     - `createLogBuffer(data: Record<string, unknown>): Buffer | undefined`: Creates a log buffer from a data object.
     - `decodeLogBuffer(buffer: Buffer): LogMessageType | undefined`: Decodes a log buffer into a `LogMessageType`.
 
+7. **Service-Channel Contract**
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 7; Style: 1/1/1

(MD029, ol-prefix)